Project-Bootcamp/views/character-view.js

```diff
@@ -0,0 +1,78 @@
+var React = require ('react');
+var _ = require ('lodash');
+var $ = require ('jquery');
+var Button = require ('../components/button');
+var Input = require ('../components/input');
+var List = require ('../components/list');
+var BlizzardAPI = require ('../querys/query');
+
+var CharacterView = React.createClass ({
+
+    getInitialState: function () {
+
+        return {
+            character: {}
+        }
+    },
+
+    render: function () {
+
+        return (
+            <div>
+                <div>
+                    <h3><u>Character basic information form</u></h3>
+                    <p>Fill the form with the character name and realm.</p>
+                    <Input label='Name' type="text" onChange={this.handleInputChange.bind (this, 'name')} />
+                    <Input label='Realm' type="text" onChange={this.handleInputChange.bind (this, 'realm')} />
+                    <Button {...this.getButtonProps()} />
+                </div>
+                <div>
+                    <h3><small>Basic Information</small></h3>
+                    {this.renderCharacterThumbnail()}
+                    <List items={this.getListItems()} />
+                </div>
+            </div>
+        )
+    },
+
+    handleInputChange: function (type, event) {
+
+        var newState = {};
+
+        if  (event.target.value){
+            newState[type] = event.target.value;
+            this.setState(newState);
+        }
+    },
+
+    getButtonProps: function () {
+
+        return {
+            className: 'btn btn-info',
+            onClick: this.handleCharacterRequest,
+            value: 'Character status'
+        }
+    },
+
+    handleCharacterRequest: function () {
+
+        BlizzardAPI.CharacterInfo(this.state.realm, this.state.name, this.refreshCharacterInfo);
+    },
+
+    refreshCharacterInfo: function (character) {
+
+        this.setState({character: character});
+    },
+
+    renderCharacterThumbnail: function() {
+
+        return <img  className="img-thumbnail" src="???" alt="Character thumbnail">;
+    },
+
+    getListItems: function () {
+
+        return null;
```

Comment:
I suppose this is on development and its purpose is to be here temporarily, so in cases like this (and when you leave a ``console.log`` statement for testing purposes, for example) you should leave a comment with a TODO, something like:
``// TODO: Update logic to map listItems prop once real data is coming.``
It's really important to keep track on updates, code refactoring, pending work, etc.